controllers/all.controllers.js

```diff
@@ -1,4 +1,8 @@
-const { fetchAllParks, fetchArticleByID } = require("../models/topics.models"); // import model
+const { fetchAllParks } = require("../models/topics.models"); // import model
```

Comment:
You've named a model fetchAllParks here which looks like a copy and paste from a previous repo, so remember to change this 👍